DESCRIPTION

```diff
@@ -48,6 +48,7 @@ Imports:
     mvtnorm,
     future
 Suggests:
+    cmdstanr,
```

Comment:
Now we have `cmdstanr` in both depends and suggests. I think we would need some extra work to make it only used conditionally throughout if we use it only in suggests
@mattsecrest